### PR TITLE
[FSDE-31345] Add support for new empty slot in data table

### PR DESCRIPTION
**Pull request by ismael-teixeira**
https://ctl.atlassian.net/browse/FSDE-31345

### CODE REVIEW

File: src/chi-vue/src/components/data-table/DataTable.tsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ import { printElement } from '../../utils/utils';
 import { ColumnResize } from './utils/Resize';
 import Tooltip from '../tooltip/tooltip';
 import { Component, Vue } from '@/build/vue-wrapper';
+import DataTableEmpty from './DataTableEmpty';
```
Comment (jllr) [inline src/chi-vue/src/components/data-table/DataTable.tsx:60]:
```suggestion
import DataTableEmptyActionable from './DataTableEmptyActionable';
```

Comment (jllr) [reply src/chi-vue/src/components/data-table/DataTable.tsx:60]: @ismael-teixeira remeber to import the agreeded naming convention to all the cases

File: src/chi-vue/src/components/data-table/DataTable.tsx [hunk 2]
```diff
@@ -68,6 +69,11 @@ export default class DataTable extends Vue {
   @Prop() data!: DataTableData;
   @Prop() config!: DataTableConfig;
 
+  @Emit(DATA_TABLE_EVENTS.EMPTY_ACTIONABLE)
```
Comment (jllr) [inline src/chi-vue/src/components/data-table/DataTable.tsx:72]:
```suggestion
  @Emit(DATA_TABLE_EVENTS.EMPTY_ACTIONABLE_LINK)
```

Comment (jllr) [reply src/chi-vue/src/components/data-table/DataTable.tsx:72]: @ismael-teixeira same here as the previous comment

File: src/chi-vue/src/constants/classes.ts [hunk 3]
```diff
@@ -94,6 +94,7 @@ export const DATA_TABLE_CLASSES = {
   ROW_CHILD: 'chi-data-table__row-child',
   ROW_GRAND_CHILD: 'chi-data-table__row-grand-child',
   EMPTY: 'chi-data-table__row-empty',
+  EMPTY_ACTIONABLE: 'actionable',
```
Comment (jllr) [inline src/chi-vue/src/constants/classes.ts:97]:
```suggestion
  EMPTY_ACTIONABLE: '-actionable',
```

**Review by josecarlosgonzalezv — APPROVED** [review]

**Pull request merged**